tools/Utilities/HostsUtility/Views/HostsFileEditorSettingsWindow.xaml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<windowex:WindowEx
+    x:Class="DevHome.HostsFileEditor.Views.HostsFileEditorSettingsWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.HostsFileEditor.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+    xmlns:ui="using:CommunityToolkit.WinUI"
+    xmlns:common="using:DevHome.Common.Windows"
+    xmlns:windowex="using:WinUIEx"
+    MinWidth="400"
+    MinHeight="480"
+    mc:Ignorable="d">
+    <Window.SystemBackdrop>
+        <MicaBackdrop />
+    </Window.SystemBackdrop>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <common:WindowTitleBar
+            x:Name="AppTitleBar"
+            Title="{x:Bind WindowTitle}"
+            IconSource="ms-appx:///Assets/HostsUILib/Hosts.ico"
+            Grid.Row="0" />
+
+        <StackPanel
+            Padding="60,0"
+            Grid.Row="1">
+            <TextBlock
+                x:Uid="Hosts_Settings_Title"
+                Padding="6,8"
+                Style="{ThemeResource SubtitleTextBlockStyle}"
+                MaxWidth="{ThemeResource MaxPageContentWidth}"
+                Margin="{ThemeResource ContentPageMargin}"/>
+                <ScrollViewer VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Auto" >
+                    <StackPanel>

```

Comment:
Can you set `MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}"` on this StackPanel, so you don't have to set it on each control below?

Reply:
Yep, makes sense.